addons/pos_self_order/static/src/overrides/components/receipt_header/receipt_header.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="pos_self_order.ReceiptHeader" t-inherit="point_of_sale.ReceiptHeader" t-inherit-mode="extension">
+        <xpath expr="//h1[hasclass('tracking-number')]" position="after">
+            <div t-if="props.data.pickingService" class="picking-service text-center">
+                <span t-if="props.data.pickingService == 'table'" >Service at Table</span>
+                <span t-else="">Pickup At Counter</span>
+                <br/>
+                <br/>
+            </div>
+            <div t-if="props.data.tableTracker" class="table-tracker text-center">
```

Comment:
what's the use of the `table-tracker` class?

Reply:
useless it is just to detect it and maybe make test in the future but I can remove it if you want